testing/scenario_app/gradle-home/.vpython

```diff
@@ -0,0 +1,11 @@
+#
```

Comment:
I think this needs to go in the location where the .py is but we can give it a try with led to see if this location works.

Reply:
Based on https://chromium.googlesource.com/infra/infra/+/refs/heads/master/doc/users/vpython.md#common-specification:

> If an individual or embedded specification cannot be found, vpython will probe walk filesystem towards root (or .gclient root) looking for a common specification file. This file must be named .vpython and be located in or above the directory of the invoked script.